### DISCUSSION TITLE
Adding Lucas Cimon as a py-pdf Github Organization owner

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/py-pdf.github.io#3] [General]
Hi everybody,

I recently got into contact with [Lucas Cimon](https://github.com/Lucas-C). He is the maintainer of [fpdf2](https://pypi.org/project/fpdf2/), a Python library for creating PDF documents.

I would like to add him as an owner of the py-pdf GitHub organization:

* **Trust**: Lucas is actively maintaining fpdf2 for many years. The project seems to do very well, with a healthy community. So I trust in his project management skills and that he has his security under control. I had a few conversations via e-mail with him; all of them were super pleasant. So I also have trust in him when it comes to the community building aspect of maintaining open source projects.
* **Scope**: With Lucas joining the py-pdf organization, we can widen our scope. Instead of being so focused on pypdf, we can actually make it a Python/PDF organization. I hope that is to the benefit of the whole ecosystem. fpdf2 can create new PDF documents from scratch - something that is very hard to do with pypdf. If you're curious about the discussion of fpdf2 joining the py-pdf organization, see [Plans for fpdf2 to join the py-pdf org](https://github.com/PyFPDF/fpdf2/discussions/752)
* **Exchange of ideas**: Sometimes we are stuck. There might be library design questions, uncertainties about the PDF standard, or testing difficulties. Having more people who we can contact is great :-)
* **Rules**: Lucas and me agreed on some [rules for py-pdf organization owners](https://py-pdf.github.io/py-pdf-owners.html). I tried to write those rules in a way that could also apply for new owners we might want to add. I feel a bit like [Sheldon](https://youtu.be/-qTG4llvt1g?t=42) writing this, but it's better to agree on this beforehand :smile: 
* [**Bus-Factor**](https://en.wikipedia.org/wiki/Bus_factor): Last but not least, I want to ensure that we will never have the problem again that the maintainer of pypdf (currently me) becomes unavailable and the whole (active) community suffers. By having another Github organization owner, this becomes way more unlikely.

I've already added him as a member to the organization, but I will wait until 30th of April 2023 (1 week + I'm on vacation, so a bit more) until I make him an owner. If you have doubts, please either comment or ping me (info@martin-thoma.de). Otherwise, give him a warm welcome :hugs:

**Comment by pubpub-zz** [comment]
welcome onboard!😊

**Comment by MartinThoma** [comment]
@Lucas-C You are now an owner of the py-pdf organization. Thank you for joining :hugs:

**Comment by Lucas-C** [comment]
Hello everyone! 👋

I've been a bit busy over the last few weeks, sorry for not dropping a comment there earlier.

I'm honored to be part of @py-pdf, and by Toma's trust in me 😊

I agree to abide by this comunity rules, and I'm eager to contribute there, and work with other Python charmers! 🐍

For people who don't know me, I'm a French developper, active in the Python community for around 10 years.
My GitHub page provides a bit more details: https://github.com/Lucas-C/
I'm not paid to work on Python libraries, my work here is entirely volunteer-based.

When working on [FLOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) projects I usually prefer things to be public / transparent, so the best way to exchange with me is usually through GitHUb _issues_ & _discussions_.
But if you really need to contact me directly / privately, you can drop me an email at lucas.cimon (Google email provider).
